### PR TITLE
Adds SSL to config

### PR DESCRIPTION
**Pull request by ProgramComputer**
Adds SSL to config. #2840

### CODE REVIEW

**Comment by knorrium** [comment]
@ProgramComputer heads up that I'm adding tests for the Docker config and you'll have to rebase once #3776 gets merged. Sorry for the inconvenience.

**Comment by knorrium** [comment]
@ProgramComputer we merged the other PR, can you please fix the merge conflicts so the new tests can run?

**Comment by ProgramComputer** [comment]
> @ProgramComputer we merged the other PR, can you please fix the merge conflicts so the new tests can run?

Merge conflicts fixed

**Comment by wiz** [comment]
Thanks for the PR, but could you default to SSL = false?
Since this is something new it should not be enabled by default

**Comment by ProgramComputer** [comment]
> Thanks for the PR, but could you default to SSL = false? Since this is something new it should not be enabled by default

Changed the SSL default to false

File: production/mempool-config.testnet.json [hunk 1]
```diff
@@ -22,7 +22,8 @@
   "CORE_RPC": {
     "PORT": 18332,
     "USERNAME": "__BITCOIN_RPC_USER__",
-    "PASSWORD": "__BITCOIN_RPC_PASS__"
+    "PASSWORD": "__BITCOIN_RPC_PASS__",
+    "SSL":false
```
Comment (softsimon) [inline production/mempool-config.testnet.json:26]: You can revert all production config additions, as we don't need to add when the default config is the same. Keeps them cleaner.

**Review by softsimon — CHANGES_REQUESTED** [review]

**Pull request closed without merging**